pkg/devfile/parser/util/utils.go

```diff
@@ -25,10 +25,8 @@ import (
 	"github.com/hashicorp/go-multierror"
 )
 
-// Default filenames for create devfile
-const (
-	OutputDevfileYamlPath = "devfile.yaml"
-)
+// Contains common naming conventions for devfiles to look for when downloading resources
+var DevfilePossibilities = [...]string{"devfile.yaml", "devfile.yml", ".devfile.yaml", ".devfile.yml"}
```

Comment:
I think this sorting is different than the one mentioned in https://github.com/devfile/devfile-web/blob/main/libs/docs/src/docs/no-version/library.md#procedure, when we first added the parser support of the `.y(a)ml` (PR https://github.com/devfile/library/pull/186/files). I think the correct sort should be:

```suggestion
var DevfilePossibilities = [...]string{"devfile.yaml", ".devfile.yaml", "devfile.yml", ".devfile.yml"}
```

Reply:
Thanks for pointing that out. For the functionality it doesn't change anything but could improve search performance slightly if that is the order we want to search in (and if they are most commonly named in that order). See my last commit for the change :)